ansible.cfg

```diff
@@ -0,0 +1,496 @@
+# config file for ansible -- https://ansible.com/
+# ===============================================
```

Comment:
is it worth keeping all this boilerplate in tree?  I imagine it evovles with each version of ansible, so might as well just refer to ansible docs?

Reply:
Meant to remove that. :)